src/sentry/db/models/fields/hybrid_cloud_foreign_key.py

```diff
@@ -45,21 +45,25 @@
 from __future__ import annotations
 
 from enum import IntEnum
+from typing import Any, Callable, Tuple
 
+from django.apps import apps
 from django.db import models
 
 __all__ = ("HybridCloudForeignKey",)
 
-from typing import Any, Tuple
-
-from django.apps import apps
-
 
 class HybridCloudForeignKeyCascadeBehavior(IntEnum):
     CASCADE = 1
     SET_NULL = 2
```

Comment:
@getsentry/hybrid-cloud this enum is unused outside this file -- should it be removed?

and/or can you make sure the change to this file is ok -- it is the only manual change of this PR